src/style.css

```diff
@@ -0,0 +1,159 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: center;
+  padding-bottom: 80px;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 30px 15px 15px;
+  margin: 50px 24px 0;
+}
+
+.picture {
+  display: flex;
+  align-items: center;
+  max-width: 158px;
+  margin: 0 auto 45px;
+}
+
+.picture img {
+  max-width: 100%;
+  height: auto;
+}
+
+.item-name {
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  margin: 0 0 6px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.rate {
+  margin-top: 15px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.star {
+  margin-right: 4px;
+  height: 15px;
+  width: 15px;
+}
+
+.price-block {
+  margin-top: 21px;
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.price {
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 19px;
+}
+
+.text {
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  font-size: 11px;
+  line-height: 13px;
+  font-weight: normal;
+}
+
+.buy-button {
+  font-family: "Roboto", sans-serif;
+  display: block;
+  border: 1px solid #00acdc;
+  text-decoration: none;
+  background: #00acdc;
+  border-radius: 5px;
+  font-weight: bold;
+  font-size: 14px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  line-height: 16px;
+  padding: 12px 0;
+}
+
+.buy-button:hover {
+  color: #00acdc;
+  background-color: #fff;
+}
+
+header {
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+}
+
+nav a {
+  padding: 23px 0;
+  margin-left: 20px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```

Comment:
`font-style` and `font-family` are inherited. 
Better to put them into `body` and you don't need to duplicate them.